ravenframework/Optimizers/GeneticAlgorithm.py

```diff
@@ -158,18 +160,31 @@ def getInputSpecification(cls):
                                                                         \item bitFlipMutator.
                                                                       \end{itemize}
                                                                     \end{itemize}
-                                                \item survivorSelectors:
+                                                  \item survivorSelectors:
                                                                       \begin{itemize}
                                                                         \item ageBased.
                                                                         \item fitnessBased.
                                                                       \end{itemize}
+                                                  \item constraintHandling:
+                                                                    \begin{itemize}
+                                                                      \item hard.
+                                                                      \item soft.
+                                                                    \end{itemize}          
                                                 \end{itemize}""")
```

Comment:
Constraint Handling is not a part of the input. We cannot have it in the description.

Reply:
Constraint Handling is now commented and left for future improvement.